packages/trader/src/App/app.jsx

```diff
@@ -15,34 +15,26 @@ const TradeModals = Loadable({
     loading: () => null,
 });
 
-class App extends React.Component {
-    constructor(props) {
-        super(props);
-        const {
-            passthrough: { WS, root_store },
-        } = props;
-        this.root_store = initStore(root_store, WS);
-    }
+const App = ({ passthrough }) => {
+    const root_store = initStore(passthrough.root_store, passthrough.WS);
```

Comment:
should we run this on every render?

Reply:
I don't think we actually should. thanks for bringing this up, I've created a state out of it to run it component initialization.